acceptance/indy_acceptance/indy-tests/libraries/logger.py

```diff
@@ -0,0 +1,88 @@
+"""
+Created on Nov 22, 2017
+
+@author: nhan.nguyen
+
+Containing classes to catch the log on console and write it file.
+"""
+
+import sys
+import os
+import time
+import errno
+import logging
+from .result import Status
+from .constant import Colors
+
+
+class Printer(object):
```

Comment:
Any class extends `object`. So there is no need to specify `object` as a superclass explicitly.

Reply:
Updated. Please refer to the latest code.